README.md

```diff
@@ -256,8 +291,8 @@ If you have something you'd like to contribute to jekyll-polyglot, please open a
 
 
 ## 2.0 Roadmap
-* [ ] - **site language**: portuguese Brazil `pt-BR` `pt-PT`
-* [ ] - **site language**: portuguese Portugal `pt-BR` `pt-PT`
+* [x] - **site language**: portuguese Brazil `pt-BR`
```

Comment:
_NICE_ thank you @george-gca for this excellent contribution and readme cleanup!